gen/README.md

```diff
@@ -4,14 +4,16 @@ GDAL.jl generation documentation
 ### Install Clang.jl
 Because a few tricks are used to be able to properly write documentation,
 this fork of Clang.jl is used: https://github.com/visr/Clang.jl (branch gdal)
+For the generation process `julia 0.5-dev` based on `LLVM 3.7.1` and with
+`BUILD_LLVM_CLANG=1` in `Make.user` is used.
 
 ### Make sure you have the GDAL header files
-The `wrap_gdal.jl` script looks in `/usr/local/include` by default.
+The `wrap_gdal.jl` script finds them in the directory set in the `gdalpath` variable.
 
 ### Prepare the GDAL Doxygen XML
 1. In the GDAL SVN, run Doxygen with `GENERATE_XML = YES`
-2. In the `xml` output dir, run `combine.xslt` to combine all documentation in one XML
-3. Copy this file to `GDAL.jl/gen/all.xml`
+2. Follow the instructions in `combine_gdal_doxygen_xml.py`
+3. Copy the result to `GDAL.jl/gen/doxygen.xml`
```

Comment:
Yup, I realized and did check. It was when I thought it's probably better that we find ways of making it easier for subsequent developers to go through the process of regenerating the documentation. Thanks again for the instructions! I wouldn't have made it that far without them.